pkg/faketikv/cluster.go

```diff
@@ -16,65 +16,39 @@ package faketikv
 import (
 	"context"
 	"fmt"
-	"math/rand"
-	"sort"
 
 	"github.com/juju/errors"
 	"github.com/pingcap/kvproto/pkg/metapb"
 	"github.com/pingcap/pd/pkg/faketikv/cases"
-	"github.com/pingcap/pd/pkg/faketikv/simutil"
-	"github.com/pingcap/pd/server/core"
 )
 
 // ClusterInfo records all cluster information.
 type ClusterInfo struct {
-	conf *cases.Conf
-	*core.RegionsInfo
+	conf  *cases.Conf
 	Nodes map[uint64]*Node
 }
 
 // NewClusterInfo creates the initialized cluster with config.
 func NewClusterInfo(pdAddr string, conf *cases.Conf) (*ClusterInfo, error) {
 	cluster := &ClusterInfo{
-		conf:        conf,
-		RegionsInfo: core.NewRegionsInfo(),
-		Nodes:       make(map[uint64]*Node),
+		conf:  conf,
```

Comment:
we may remove `ClusterInfo`, the Drive also know the `conf` and the `Conn`.

Reply:
+1. But may be next step.